src/Response/Struct/OfficeCollection.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\EcontBundle\Response\Struct;
+
+use function count;
+use Countable;
+use IteratorAggregate;
+use Webmozart\Assert\Assert;
+
+/**
+ * Only Address type 1 supported
+ *
+ * @url https://api.speedy.bg/web-api.html#href-ds-shipment-address
+ */
+class OfficeCollection implements Countable, IteratorAggregate
+{
+    /**
+     * @var Office[]
+     */
+    private array $offices;
+
+    public function __construct(array $offices)
+    {
+        Assert::allIsInstanceOf($offices, Office::class);
+
+        $this->offices = $offices;
+    }
+
+    /**
+     * @return Office[]
+     */
+    public function getIterator(): iterable
+    {
+        foreach ($this->offices as $key => $office) {
+            yield $key => $office;
+        }
+    }
+
+    public function get($key): ?Office
```

Comment:
Maybe we should index offices by their identifier? That would make more sense for the `get` method